sympy/physics/mechanics/kane.py

```diff
@@ -244,11 +295,12 @@ def _initialize_constraint_matrices(self, config, vel, acc):
 
         Parameters
         ==========
-        config : iterable of SymPy expressions
+
+        config : iterable of expressions
             The configuration constraint expressions.
-        vel : iterable of SymPy expressions
+        vel : iterable of expressions
             The velocity constraint expressions.
-        acc : iterable of SymPy expressions
+        acc : iterable of expressions
```

Comment:
Out of curiosity why is "Sympy" being removed here (and elsewhere in similar situations)?